src/util.rs

```diff
@@ -431,6 +435,86 @@ pub fn compile_cdn_list(cdn_file_path: Option<String>) -> Vec<String> {
     return_vec
 }
 
+fn sc_extract_exact_timestamps(html_fragment: &Html) -> Result<ExtractedTimestamps> {
+    let exact_dt_selector = match Selector::parse("div > div[data-requests]") {
+        Ok(s) => s,
+        Err(_) => Err(DeriveDateError::SelectorError)?,
+    };
+
+    match html_fragment
+        .select(&exact_dt_selector)
+        .nth(0)
+        .ok_or(DeriveDateError::ScraperElementError)
+    {
+        Ok(d) => {
+            match d
+                .value()
+                .attr("data-requests")
+                .ok_or(DeriveDateError::ScraperAttributeError)
+            {
+                Ok(s) => {
+                    // Parse the clips_json into the struct StreamsChartsTwitchClip with serde_json
+                    let clips_payloads: Vec<StreamsChartsTwitchClip> =
+                        serde_json::from_str(&s.to_string()).unwrap();
```

Comment:
I think we don't have to convert ```s``` to string and back to &str here, since it's already a &str, can just leave it as ```serde_json::from_str(s).unwrap()```. Not super important, just nicer! :)